geomesa-raster/src/main/scala/org/locationtech/geomesa/raster/AccumuloStoreHelper.scala

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright 2014 Commonwealth Computer Research, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.locationtech.geomesa.raster
+
+import java.io.Serializable
+import java.util.{Map => JMap}
+import javax.imageio.spi.ServiceRegistry
+
+import org.apache.accumulo.core.client.mock.{MockConnector, MockInstance}
+import org.apache.accumulo.core.client.security.tokens.PasswordToken
+import org.apache.accumulo.core.client.{Connector, ZooKeeperInstance}
+import org.locationtech.geomesa.security._
+
+import scala.collection.JavaConversions._
+
+//TODO: WCS: refactor this and AccumuloDataStoreFactory to remove duplication, etc...
```

Comment:
Good point.  I'd be happy to see this happen on either side of the merge at this point.